design/IDL-Motoko.md

```diff
@@ -312,26 +312,14 @@ just suggestions.
   mapping `i`, will generate an IDL type `e(t)` as in the previous point, and
   and check that `e(t) <: t_spec` (using IDL subtyping).
 
-* Converting IDL types to Motoko types
+* Importing IDL types from the Motoko compiler
 
-  If `foo.did` a textual IDL file, then
+  If `path/to/foo.did` a textual IDL file, then a declaration
 
-      didc foo.did -o foo.mo
+      import Foo "path/to/foo"
```

Comment:
This is not supported yet?

Reply:
Yeah, but it's an one design doc and thus was already there